geopatterns_demo/random_phrase.py

```diff
@@ -0,0 +1,6 @@
+import loremipsum
+
+
+def make_random_text():
+    """Method create random phrase using loremipsum library."""
+    return {'text': loremipsum.generate(1, 'short')}
```

Comment:
This function should return:

- either just a `str`,
- or a `RandomPhrase` instance.

I suggest the first option because it makes the function very simple. Simpler is better.